samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_temp.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2019 Foundries.io
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#define LOG_MODULE_NAME app_lwm2m_temp
+#define LOG_LEVEL CONFIG_APP_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include <zephyr.h>
+#include <sensor.h>
+#include <net/lwm2m.h>
+
+/* use 25.5C if no sensor available */
+static struct float32_value temp_float = { 25, 500000 };
+static struct device *die_dev;
+static s32_t timestamp;
+
+#if defined(CONFIG_TEMP_NRF5_NAME)
+static int read_temperature(struct device *temp_dev,
+			    struct float32_value *float_val)
+{
+	__unused const char *name = temp_dev->config->name;
```

Comment:
What's `__unused`?

Reply:
Woops.  Removing __unused.  It's clearly used :P